### ISSUE TITLE
reordering the packets

### ISSUE THREAD

**Issue by spidermanja** [teragrep/rlp_02#2]
When Responses comes in the different order from the java-relp-server, this might need  to sort and reorganize the packets for proper flow control. 
find the attachment for anomalies
![rlp_2closeanomalies1](https://user-images.githubusercontent.com/93120208/201666353-191b3ba7-2316-4078-b5cd-082906a17a9d.png)

**Comment by kortemik** [comment]
Hi,

TCP might re-order the packages but that is ok because network layer will
correct the order and the application does not see tcp level order but only
the order that the packets were sent with. Relp also accepts re-ording of
responses from the server to the client but that's different thing but
still possible. I checked the java-relp-server code and it should sent
everything in the order that the requests were received.

**Comment by spidermanja** [comment]
Handling  closeResponse for null currently fix the issue. 
However, potential security related issue 
- Packet with invalid error status possibly generate NULL deference. 
Possible vulnerability,  A NULL pointer dereference might occur when the application dereferences a pointer that it expects to be valid, but is NULL, typically causing a crash or exit.

**Comment by kortemik** [comment]
Hi,

Would it be possible to check if it is NULL before accessing it and
initialize the get method to return empty string in case it is null?

Kind regards,
Mikko

**Comment by spidermanja** [comment]
Yes, adjusted on this commit 3e5053708d231444099cf38362acc9705a8d2357
but without returning any string.
Seems no more complains now👍
